### PR TITLE
Update examples

### PR DESCRIPTION
**Pull request by DanielZhangQD**
### What problem does this PR solve?

Update examples according to the feedback from oncall issues
### What is changed and how does it work?

1. Replace `tidb-scheduler` with `default-scheduler`
2. Disable failover by default
### Code changes

- [ ] Has Go code change
- [ ] Has CI related scripts change

### Tests

- [ ] Unit test 
- [ ] E2E test 
- [ ] Manual test 
- [ ] No code 

### Side effects

- [ ] Breaking backward compatibility 
- [ ] Other side effects: 

### Related changes

- [ ] Need to cherry-pick to the release branch 
- [ ] Need to update the documentation 

### Release Notes

Please refer to [Release Notes Language Style Guide](https://github.com/pingcap/tidb-operator/blob/master/docs/release-note-guide.md) before writing the release note.

```release-note

```

### CODE REVIEW

File: examples/advanced/tidb-monitor.yaml [hunk 1]
```diff
@@ -375,10 +375,10 @@ spec:
     ## Base image of the initializer.
     baseImage: pingcap/tidb-monitor-initializer
     ## ImagePullPolicy of initializer.
-    imagePullPolicy: IfNotPresent
+    # imagePullPolicy: IfNotPresent
     ## Initializer version.
-    version: v5.2.0
-    
+    version: v5.2.1
```
Comment (csuzhangxc) [inline examples/advanced/tidb-monitor.yaml:380]: How about updating this version until the next release with our shell command?

Comment (DanielZhangQD) [reply examples/advanced/tidb-monitor.yaml:380]: Yes, this is the version that is missed in the last update. Now all the versions are v5.2.1 and we can update to v5.3.0 in the next release.

**Review by csuzhangxc — APPROVED** [review]

**Review by KanShiori — APPROVED** [review]

**Pull request merged**